### DISCUSSION TITLE
注釈をマウスオーバーである程度読めるように

### DISCUSSION THREAD

**Discussion by Phroneris** [increments/qiita-discussions#321] [要望・アイデア・不具合]
- QiitaのMarkdownパーサー変更にともなう、脚注についての書き方と注意点 - Qiita
  https://qiita.com/phigasui/items/010f4c79ff47bbaaebdd

上記記事の[コメント欄](https://qiita.com/phigasui/items/010f4c79ff47bbaaebdd#comments)でもやり取りがされていますが、注釈の内容がクリックやスクロールをしないと丸っきり読めなくなってしまったのはやはり不便です。
些細な閑話をマウスオーバーでサクッと読むことが想定されていたであろう過去の記事を読む際などは特に、現状の仕様だといちいち行きつ戻りつを強いられるのでとても快適には読めません。

少なくとも従来のテキスト+装飾程度の内容であれば、マウスオーバーで読めるようにしていただきたいです。
じゃあ複数行も…それならリストも…いっそ全て…とか考え始めてしまうと程度問題になるのは想像に難くないですが、それらはまぁ未来に期待するとして（期待してます！！）、ひとまずこのくらいの後方互換性はあってほしいです。

**Comment by tanaka-asami415** [comment]
@Phroneris 
ご意見ありがとうございます🙏

記事のコメントの繰り返しになってしまいますが、
Markdownパーサー変更にともない、ベースとなっている新しいMarkdownパーサーの仕様にできる限り準拠する方針となったため、
注釈文が表示されなくなりました。

ですが、ご要望が多ければ改めて検討したいと考えておりますので
こちらのDiscussions等で、他のユーザーさまからのご意見もお待ちして検討したいと思います🙇

**Comment by esperecyan** [comment]
ポップアップする脚注があるというのは、Qiitaを利用するメリットの中でも大きなもののひとつでした。
復旧を強く希望します。
